pkgs/development/libraries/fontconfig/default.nix

```diff
@@ -74,7 +74,7 @@ stdenv.mkDerivation rec {
 
   buildInputs = [
     expat
-  ] ++ stdenv.lib.optional stdenv.isDarwin CoreFoundation;
+  ] ++ lib.optional stdenv.isDarwin CoreFoundation;
```

Comment:
Thanks for taking care of this! I might be wrong, but aren't you missing "lib" as parameter at the top of the file (first pair of curlies?, Sorry, I don't know how it is called …)

Like so:
```
{ stdenv
, lib
…
}
```

Reply:
Everything is allright now in the `staging` branch (see https://github.com/NixOS/nixpkgs/blob/staging/pkgs/development/libraries/fontconfig/default.nix). Probably there was a merge of different commit or something like that.

Reply:
Oh, ok, that make sense, since the other stdenv.lib got probably fixed there.